.github/workflows/build.yml

```diff
@@ -13,9 +13,9 @@ jobs:
       fail-fast: false
       matrix:
         java-version:
-          - 8
-          - 9
-          - 10
+          #- 8
+          #- 9
+          #- 10
```

Comment:
I'll address that in follow up

Reply:
Even though I don't have a clear plan right now.

They are commented out because the android version we updated to doesn't wanna run on java 10 and below